th/2017/winter.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="fr">
+	<head>
+		<title>ฤดูหนาว 2017 | ไทย | The Fansub Database</title>
```

Comment:
for anime, Season information reference from  https://myanimelist.net/anime/season/2017/winter

In Thailand, Anime which it was aired on December (Eg. Kochinpa Dainishi) We also count it as Winter 2017.  So can I still use Winter 2017?